### ISSUE TITLE
wimp plugin - search function not working

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#102]
```
What steps will reproduce the problem?
1.hitting the search Button returtns a blank screen.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
Squeezer 1.1.0 beta 2
Android 4.4.4 on a Nexus 7

Please provide any additional information below.

```

Original issue reported on code.google.com by `joachim.fishman` on 2014-08-26 13:21:53

**Comment by nikclayton** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer

**Comment by kaaholst** [comment]
Fixed in 2.0.0
